infrastructure/terraform/05-data-sources.tf

```diff
@@ -1,6 +1,6 @@
 data "azurerm_key_vault" "infra_key_vault" {
   name                = "hmi-shared-kv-${var.environment}"
-  resource_group_name = "hmi-sharedinfra-${var.environment}-rg"
```

Comment:
This is dynamic for each environment already, so why do we need to split it into a variable?

Reply:
stg and prod are sharedservices, while sbox, dev, test are sharedinfra

Reply:
Staging and Production should be moved over as well to Shared Infra as well.